Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesHii.h

```diff
@@ -51,6 +51,7 @@ typedef struct {
   // Feature data
   BOOLEAN            LockBios;
   BOOLEAN            SmmBwp;
+  BOOLEAN            EnableWifiBT;

```

Comment:
Please add new entries at the end of the structure @Mixss

Reply:
This is already done.